doc/machines/index.rst

```diff
@@ -38,28 +40,47 @@ Parameters:
 
 ``user`` (optional, default=root)
 
+``service_id`` (required)
+
 When the SSH token type is assigned to a client, the user specified in the
 user parameter
 can login with the private key of the SSH token.
 
-In the ``sshd_config`` file you need to configure the ``AuthorizedKeysCommand``.
-Set it to::
+The ``service_id`` identifies the SSH servers or group of SSH servers, where the login is allowed to occur.
+
+To facilitate this, the SSH server fetches the managed SSH keys from the privacyIDEA server on demand.
+The SSH server uses the ``AuthorizedKeysCommand`` in the ``sshd_config`` to do this.
+
+There is an Python script `privacyidea-authorizedkey` in the privacyideaadm repository. Note, that this
+script currently does not support the ``service_id``.
+The `tools/` directory of the privacyIDEA Server ships a shell script `privacyidea-authorizedkeys` that
+supports the ``service_id``.
+
+In the ``sshd_config`` file you need to configure the ``AuthorizedKeysCommand`` accordingly.
+Set it to e.g.::
 
    privacyidea-authorizedkeys
 
-This will fetch the SSH public keys for the requesting machine.
+This will fetch the SSH public keys for the requesting machine and the given user.
+
+If you are using the shell script you need to configure the privacyIDEA Server and
+the service account at the top of the script.
 
-The command expects a configuration file
+The Python script however expects a configuration file
 */etc/privacyidea/authorizedkeyscommand* which looks like this::
 
    [Default]
    url=https://localhost
    admin=admin
    password=test
    nosslcheck=False
+   service_id=webservers
```

Comment:
But the python script does not (yet) support the `service_id`?

Reply:
You are right. I am currently not quite sure, how we should handle the scripts.

The script in this repo in tools/ is nice, since it is a shell script with less depedencies.

The script in the other repo is more sophisticated with a config file, but needs dependencies.